creator-node/src/services/sync/processSync.js

```diff
@@ -510,7 +513,6 @@ async function processSync(
         logger.info(redisKey, 'Saved all AudiusUser entries to DB')
 
         await transaction.commit()
-        await redisLock.removeLock(redisKey)
```

Comment:
couldn't figure out why we have this redundant removeLock call - the one in `finally` block should cover